tests/unit/amo/helpers.js

```diff
@@ -219,23 +219,14 @@ export function dispatchClientMetadata({
 export function dispatchSignInActions({
   authToken = userAuthToken(),
   userId = 12345,
-  username = 'user-1234',
-  // eslint-disable-next-line camelcase
-  display_name = null,
-  permissions = [],
+  user = {},
```

Comment:
I find this confusing because it looks like the caller is forgetting to use a base object. Take this as an example:

```js
     const { store } = dispatchSignInActions({
       user: { permissions: [COLLECTIONS_EDIT] },
     });
```

It looks like the caller accidentally forgot to do:

```js
     const { store } = dispatchSignInActions({
       user: createUserAccountResponse({ permissions: [COLLECTIONS_EDIT] }),
     });
```

But I know it's a useful shortcut. Maybe it would help just to rename the argument?

```js
     const { store } = dispatchSignInActions({
       userProps: { permissions: [COLLECTIONS_EDIT] },
     });
```

Either way, I like the new approach to `dispatchSignInActions()`.

Reply:
Oh, that's a good point. Yeah, I'll change it to `userProps`, good call. 👍